openedx/core/djangoapps/oauth_dispatch/docs/decisions/0011-major-versioning.rst

```diff
@@ -0,0 +1,101 @@
+11. Handling Breaking Changes with Major Versions
+=================================================
+
+Status
+------
+
+Draft
+
+Temporary Implementation Notes
+------------------------------
+
+This idea is being put on hold because it is hard to implement.  Here are some notes
+about the implementation in case we ever decide to revive this:
+
+JWT Changes:
+- sub: changing meaning (does anything care internally?)
+- username: must add to scope requested and authorized.  where and how?
+- profile: retire? leave alone to shrink our problems?
+- copy role out of profile - shouldn't require a major version change.
+
+Places to encapsulate JWT details:
+- frontend-auth library
+- ... ?
+- Note: The fewer pieces of code that care what the JWT looks like, the better.
+
+Django Settings:
+    Retire:
+    - JWT_SUPPORTED_VERSION
```

Comment:
@christopappas: Here is where I was looking into adding Major Version support, and I was noting that any other checks should be retired.  Major Version support ended up being too big a lift, but I still think `JWT_SUPPORTED_VERSION` should be retired.